changelogs/fragments/controller-python-warning.yml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
```

Comment:
Should this be higher than `minor` so it's more prominent in the changelog?

Reply:
The warning itself is minor

Reply:
The policy change is major. The changelog & porting guides are reviewed by people *before* upgrading. Let's make sure we over-communicate these changes.